docs/topics/development/testing.rst

```diff
@@ -80,9 +80,31 @@ the respective docker image, use the following command::
 
     docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml up -d
 
+To make sure the appropriate containers are setup run the following commands:
+
+Setup for ```worker``` container::
+
+    docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml exec worker make -f Makefile-docker update_deps update_assets
+
+Followed by::
+
+    docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml restart worker
+
+Setup for ```web``` container::
+
+    docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml exec web make -f Makefile-docker update_deps update_assets
+
+Followed by::
+
+    docker-compose -f docker-compose.yml -f tests/ui/docker-compose.selenium.yml restart web
+
+To setup the data within the database that the tests will look for run this command::
+
+    make -f Makefile-docker setup-ui-tests
+
```

Comment:
Why do you have to do `make -f Makefile-docker`?  make is written to use `Makefile-docker` within the docker containers.

Reply:
Hmm okay Ill confirm before changing.

Reply:
So when I run ```make setup-ui-tests``` I get an error that says ```No rule to make target 'setup-ui-tests'.  Stop.``` I know ```make initialize_docker``` exists within ```Makefile-os``` maybe that's why?

Reply:
I'm not sure where this command is expected to be run.  If you see `Makefile` - the code to identify which makefile to use expects a folder to be present to identify it's running inside the web or worker container.

Commands which are expected to run outside of docker should be in Makefile-os; commands that run inside the container, in Makefile-docker.  (We have to override this specifically when running tests on Travis because the tests aren't run inside docker containers)

Reply:
Its meant to run inside the docker container as it is executing a bunch of database commands.